src/xcode/ENA/ENAUITests/AppDelegate/ENAUITests_11_QuickActions.swift

```diff
@@ -81,7 +81,7 @@ class ENAUITests_11_QuickActions: CWATestCase {
 		XCTAssertTrue(app.segmentedControls[AccessibilityIdentifiers.ContactDiary.segmentedControl].waitForExistence(timeout: .short))
 	}
 	
-
+	/* Commented out in favour SRS story
```

Comment:
I think it still makes sense to have a test like that, but you probably need to find a new way to test through the submission instead of using a TAN